multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -2004,6 +2017,117 @@ def save(self, path: str, standalone: Optional[bool] = True):
                     f"removed."
                 )
 
+    def export_onnx(
```

Comment:
Do we need a unit test for the onnx export? Maybe some code and data from the `production/onnx_text.py`.

Reply:
Done.